server/jetstream_test.go

```diff
@@ -17508,6 +17508,25 @@ func TestJetStreamWorkQueueSourceNamingRestart(t *testing.T) {
 	}
 }
 
+func TestJetStreamDisabledHealthz(t *testing.T) {
+	s := RunRandClientPortServer()
+	if config := s.JetStreamConfig(); config != nil {
```

Comment:
Not necessary since you are not configuring a server with JetStream.

Reply:
removed